src/main/java/org/minions/devfund/damianvp/battleship/Ocean.java

```diff
@@ -0,0 +1,159 @@
+package org.minions.devfund.damianvp.battleship;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    static final int LENGTH = 20;
+    static final int TOTAL_SHIPS = 13;
+    private Ship[][] ships;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+    private static final int BATTLESHIP_NUMBER = 1;
+    private static final int BATTLE_CRUISER_NUMBER = 1;
+    private static final int CRUISER_NUMBER = 2;
+    private static final int LIGHT_CRUISER_NUMBER = 2;
+    private static final int DESTROYER_NUMBER = 3;
+    private static final int SUBMARINE_NUMBER = 4;
+    private static final Map<String, Integer> FLEET = new HashMap<>();
+    static {
+        FLEET.put("BattleShip", BATTLESHIP_NUMBER);
+        FLEET.put("BattleCruiser", BATTLE_CRUISER_NUMBER);
+        FLEET.put("Cruiser", CRUISER_NUMBER);
+        FLEET.put("LightCruiser", LIGHT_CRUISER_NUMBER);
+        FLEET.put("Destroyer", DESTROYER_NUMBER);
+        FLEET.put("Submarine", SUBMARINE_NUMBER);
+    }
+
+    /**
+     * Ocean constructor.
+     */
+    Ocean() {
+        ships = new Ship[LENGTH][LENGTH];
+        for (int i = 0; i < LENGTH; i++) {
+            for (int j = 0; j < LENGTH; j++) {
+                EmptySea emptySea = new EmptySea();
+                this.ships[i][j] = emptySea;
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+    }
+
+    /**
+     * methohd to place all ship randomly.
+     */
+    void placeAllShipsRandomly() {
+        Random random = new Random();
+//        ShipFactory creator = new ShipFactory();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)